### DISCUSSION TITLE
You've probably hit a rate limit ERROR

### DISCUSSION THREAD

**Discussion by 1nfiniteZest** [crunchy-labs/crunchy-cli#364] [Q&A]
I keep getting this error. Anyone know a fix?
![Uploading Screenshot 2024-04-03 at 11.49.06 PM.png…]()

**Comment by Ichicoro-dev** [comment]
I'm getting this too...

**Comment by transmatrix** [comment]
I got this error a few weeks ago and resolved it by updating to the latest release of crunchy-cli

**Comment by bytedream** [comment]
I'm gathering information on this one. For anyone experiencing this issue with other series, can you tell me the series + your country (this issue is region specific)?

**Reply by Ichicoro-dev** [reply]
:: √ Logged in
:: √ Parsed url 1
:: → Episode 7.5 (How to Get Stronger) of Solo Leveling season 1 is not available with en-US audio
:: → Episode 12 (A Knight Who Defends an Empty Throne) of Solo Leveling season 1 is not available with en-US audio
:: → Episode 13 (Arise) of Solo Leveling season 1 is not available with en-US audio
:: √ Loaded series information for url 1
:: Solo Leveling Season 1 (Solo Leveling (English Dub))
::      1. I'm Used to It » S01E01
::      2. If I Had One More Chance » S01E02
::      3. It's Like a Game » S01E03
::      4. I've Gotta Get Stronger » S01E04
::      5. A Pretty Good Deal » S01E05
::      6. The Real Hunt Begins » S01E06
::      7. Let's See How Far I Can Go » S01E07
::      8. This Is Frustrating » S01E08
::      9. You've Been Hiding Your Skills » S01E09
::      10. What Is This, a Picnic? » S01E10
:: An error occurred: You've probably hit a rate limit. Try again later, generally after 10-20 minutes the rate limit is over and you can continue to use the cli (Error URL Here but redacted as it probably contains an identifier token)

When I go to that url:
`{"error":"Unauthorized","reason":"missing_token"}`

**Comment by bytedream** [comment]
I fixed this in [v3.3.5](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.3.5). Crunchyroll began embedding subtitles in the file which only contained the video streams before, this was causing the false-positive rate limit detection

**Reply by Ichicoro-dev** [reply]
Thank you! I'll look forward to it in scoop. :)

**Reply by Ichicoro-dev** [reply]
@bytedream any ETA for scoop update? Sorry to bother, I am not in a place to install cargo.

**Reply by bytedream** [reply]
@WReptiles no, I have no control over scoop updates, they're done automatically by scoop itself
